packages/mui-material/src/Breadcrumbs/BreadcrumbCollapsed.js

```diff
@@ -26,27 +26,41 @@ const BreadcrumbCollapsedButton = styled(ButtonBase)(({ theme }) => ({
   },
 }));
 
-const BreadcrumbCollapsedIcon = styled(MoreHorizIcon)({
-  width: 24,
-  height: 16,
-});
-
 /**
  * @ignore - internal component.
  */
 function BreadcrumbCollapsed(props) {
+  const { components, ...otherProps } = props;
   const ownerState = props;
+  const CustomBreadcrumbCollapsedIcon =
+    components && components.Collapsed ? components.Collapsed : <React.Fragment />;
+
+  const BreadcrumbCollapsedIcon = styled(
```

Comment:
This needs to be created outside of the parent component. You can use the `as` prop for setting up the `components.Collapsed`.